decide/lic7.cpp

```diff
@@ -0,0 +1,29 @@
+#include <cassert>
+#include <decide/decide.hpp>
+
+bool lic7(Points points, Parameters parameters) {
+  int npts = points.size();
+  int kpts = parameters.KPTS;
+  double length = parameters.LENGTH1;
+
+  // 1 <= K_PTS <= (NUMPOINTS - 2). According to specification.
+  assert(1 <= kpts);
+  assert(kpts <= npts - 2);
```

Comment:
What does `assert` do here? Shouldn't the function just return `false`?

Reply:
I was unsure what to do here. The specification does not say that the function should return false in these cases. The inequalities are simply added at the bottom of the specification. I interpreted that as, these values should hold, or the result of the function is invalid.

Reply:
I think the most logical way to approach it is to return `false`.

Reply:
Since it's not mentioned in the specification, it can be classified as UB and we should be able to just return `false` if we want (or crash, as long as we motivate why in the doc-comment).

Reply:
That's a very bad practice. Imho they should be checked in the function, especially since it's trivial to do so.

Reply:
I sort of agree with you, but I don't like the fact that we implement giving an answer if the input parameters are invalid. However, I added these checks, but with a comment about it being UB.